src/components/DatePicker/form.tsx

```diff
@@ -0,0 +1,25 @@
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { useForm } from 'react-hook-form';
+
+import { StyledForm } from '../../auth/components/form';
+
+import CustomDatePicker from './index';
+
+const DatePickerTestForm: React.FC = () => {
+  const methods = useForm();
+  const onSubmit = () => console.log('test');
+
+  return (
+    <StyledForm methods={methods} onSubmit={onSubmit}>
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
+        <CustomDatePicker
+          minDate={'02/03/2000'}
```

Comment:
Tą minimalną datę myślę, że możemy zmniejszyć 😄. @vis4rd z jakiego przedziału datowego mniej więcej będziemy mieli dane?

Reply:
Najstarsze dane jakie mamy na ten moment są z 2013 i szczerze mówiąc nie sądzę żebyśmy potrzebowali starsze. Jak dla mnie 2000+ jest wystarczające

Reply:
Ta data jest wpisana na razie z dupy, to pierwszy draft datepickera żeby tylko działał dla Krystiana do review xd

Reply:
@vis4rd tak, miałem na myśli zwiększyć datę xD

Reply:
@sakor88 wiadomo, że tak o, tylko wiedziałem, że jakieś dane już mamy, więc chciałem po prostu ustalić mniej więcej jaki przedział